lib/omnibus/config.rb

```diff
@@ -443,6 +443,16 @@ def reset!
     # @return [String, nil]
     default(:solaris_linker_mapfile, "files/mapfiles/solaris")
 
+    # Architecture to target when building on windows.  This option
+    # should affect the bit-ness of Ruby and DevKit used, the platform of
+    # any MSIs generated and package dlls being downloaded.
+    #
+    # See the windows_x86? and windows_x64? software definition dsl
+    # methods.
+    #
+    # @return [:x86, :x64]
+    default(:windows_arch, :x86)
```

Comment:
That would break the builds we have right now.  All our machines are 64-bit. And we still want to only be generating 32-bit builds for the foreseeable future until we get the matrix right and have other builders added.